src/libDirectoryService/MicroBlockProcessing.cpp

```diff
@@ -347,15 +346,12 @@ bool DirectoryService::ProcessMicroblockSubmissionFromShard(
   LOG_GENERAL(
       INFO, "Received microblock submission for block number " << blockNumber);
 
-  if (m_mediator.m_txBlockChain.GetLastBlock().GetHeader().GetBlockNum() <
-      blockNumber) {
+  if (m_mediator.m_currentEpochNum < blockNumber) {
```

Comment:
can change blockNumber to epochNum

Reply:
Yes, I also have thought about it, but actually the usage of blockNumber or epochNum is quite fragmental among our codebase, we can unify them in a seperate PR.

Reply:
since in this PR you changed the blocknumber to epochnumber, better change the name of variable too.

Reply:
ok